src/pygieons/ecosystem_connections.py

```diff
@@ -292,4 +292,10 @@
     {"from": "xarray-spatial", "to": "datashader"},
     {"from": "xarray-spatial", "to": "xarray"},
     {"from": "xarray", "to": "numpy"},
+    {"from": "eoreader", "to": "rioxarray"},
+    {"from": "eoreader", "to": "xarray"},
+    {"from": "eoreader", "to": "rasterio"},
+    {"from": "eoreader", "to": "geopandas"},
+    {"from": "eoreader", "to": "pystac"},
+    {"from": "eoreader", "to": "dask"},
```

Comment:
Excellent, thanks for contributing! 🙂 Would you mind adding these connections in alphabetical order and move the "eoreader" lines earlier, i.e. after [line 33](https://github.com/pygieons/pygieons/blob/main/src/pygieons/ecosystem_connections.py#L33) now having ` {"from": "eomaps", "to": "pyepsg"}`. With this, the style (i.e. alphabetical order) would remain correct. 🙏🏻